.github/workflows/build-test-measure.yml

```diff
@@ -684,6 +684,9 @@ jobs:
     steps:
       - name: Check out source files
         uses: actions/checkout@v2
+        with:
+          # By default, the fetch depth is 1. To get the last PR commit we'd need to increment that by 1.
+          fetch-depth: 2
```

Comment:
Is this still needed now that you're not getting `HEAD~`?

```suggestion
```

Reply:
Oh nope. Removing.